js/ui/status/automaticUpdates.js

```diff
@@ -0,0 +1,301 @@
+// -*- mode: js; js-indent-level: 4; indent-tabs-mode: nil -*-
+//
+// Copyright (C) 2018 Endless Mobile, Inc.
+//
+// This is a GNOME Shell component to wrap the interactions over
+// D-Bus with the eos-payg system daemon.
```

Comment:
This text is a copy & paste from the paygManager component. Please adapt it or remove it